app/Http/Controllers/FavoritesController.php

```diff
@@ -35,4 +35,14 @@ public function check(Request $request, int $cityId): bool
             ->where("city_id", $cityId)
             ->exists();
     }
+
+    public function show(Request $request): array
```

Comment:
I would prefer to return JsonResponse in controllers. I know that arrays are automatically converted into response objects, but I like to have everything on its right place.